metals/src/main/scala/scala/meta/internal/metals/ClassPath.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.internal.metals
+import scala.meta.internal.semanticdb.SymbolOccurrence
+
+object ClassPath {
+  final case class TypeSignature(value: String) extends AnyVal
+
+  def toTypeSignature(definition: SymbolOccurrence): TypeSignature = {
+    import scala.meta.internal.semanticdb.Scala._
+    @scala.annotation.tailrec
+    def qualifier(symbol: String, acc: List[String]): String = {
+      val owner = symbol.owner
+      if (owner == Symbols.RootPackage || owner == Symbols.EmptyPackage) {
```

Comment:
What about the case when `owner == Symbols.None`?

Reply:
With your next suggestion (`ownerChain`) it should be covered